### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by preetitheraja**
Simple spring boot app

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.1.6.RELEASE</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>com.springbootsimple </groupId>
```
Comment (shekhargulati) [inline pom.xml:11]: Give proper groupId

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	<artifactId>simpleapp</artifactId>
```
Comment (shekhargulati) [inline pom.xml:12]: change name

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	<version>0.0.1-SNAPSHOT</version>
+	<name>simpleapp</name>
```
Comment (shekhargulati) [inline pom.xml:14]: same as above

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+	<description>Demo project for Spring Boot</description>
```
Comment (shekhargulati) [inline pom.xml:15]: same as above

File: src/main/java/com/springbootsimple/simpleapp/SimpleappApplication.java [hunk 5]
```diff
@@ -0,0 +1,13 @@
+package com.springbootsimple.simpleapp;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SimpleappApplication {
```
Comment (shekhargulati) [inline src/main/java/com/springbootsimple/simpleapp/SimpleappApplication.java:7]: same as above

File: src/test/java/com/springbootsimple/simpleapp/SimpleappApplicationTests.java [hunk 6]
```diff
@@ -0,0 +1,16 @@
+package com.springbootsimple.simpleapp;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class SimpleappApplicationTests {
```
Comment (shekhargulati) [inline src/test/java/com/springbootsimple/simpleapp/SimpleappApplicationTests.java:10]: same as above

**Review by shekhargulati — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (preetitheraja) [reply pom.xml:11]: done

File: pom.xml (hunk 3 above)
Comment (preetitheraja) [reply pom.xml:14]: done

File: pom.xml (hunk 4 above)
Comment (preetitheraja) [reply pom.xml:15]: done

**Pull request closed without merging**